### ISSUE TITLE
Directory not found

### ISSUE THREAD

**Issue by MrGekko** [sindresorhus/grunt-php#71]
Hi the commits that fixed https://github.com/sindresorhus/grunt-php/issues/70 created a bug for me.
When I put '../' in base. The php server wont start and sais `Directory does not exit.` '..' does not work either.

**Comment by sindresorhus** [comment]
What OS and Node version?

Does `$ php -S localhost:3000 -t ../` work for you?

**Comment by MrGekko** [comment]
Windows 10
Node v0.12.7
"Does `$ php -S localhost:3000 -t ../` work for you?" no.

With version 1.5.0 of your module it works fine though options: 
 port=8080, hostname="localhost", base="../", keepalive=false, open, bin
="php", silent=false, env={}, directives={}, router="D:\\**/router.php"

**Comment by sindresorhus** [comment]
Can you try https://github.com/sindresorhus/grunt-php/commit/5b18aeeb4082c93b5db6fe07c089e7a601b05047 and see if that solves it for you?

**Comment by MrGekko** [comment]
this does start the server and reports:
Document root is *correct path*
But php is still using my grunt folder for the document root and not its parrent folder.

**Comment by MrGekko** [comment]
Do you have any idea what the problem might be?

**Comment by sindresorhus** [comment]
Not really. Submitting a failing test would help move this forward faster. I don't have time to look into this right now.

**Comment by toszter** [comment]
Try this: https://github.com/toszter/grunt-php/commit/5a687ca46572d0dff6db7d433b7855285fcb0498

**Comment by johnstonian** [comment]
@sindresorhus,

I think something is up with the npm package. I ran ``npm update`` to get grunt-php 1.5.1. However, the php.js file didn't change. Experienced the same issue as @MrGekko.

However, if I manually modify php.js to include the new lines from the latest commit, the plugin works with the base variable set at ``"../"``, or even ``"./"``

Even running ``npm install grunt-php@1.5.1`` doesn't grab the latest changes for the php.js file. It appears to be the same file from version 1.5.0. Could something be wrong with the package being served from npm?

The following are the lines I manually added to php.js file in the appropriate places to make the plugin work on my local machine:

``var path = require('path');``

and 

``args.push('-t', path.resolve(options.base));``
